pyquil/api/_config.py

```diff
@@ -40,14 +60,6 @@ class PyquilConfig(object):
         "default": "https://forest-server.qcs.rigetti.com"
     }
 
-    API_KEY = {
```

Comment:
This is not checked in Forest server.

Reply:
✝️

Reply:
fwiw the api key & user id are logged by qvm app for every request. I don't think qvm app does anything else with them besides log them, so as far as I know they're safe to remove. Just mentioning it in case something is expecting/inspecting the log messages.